.circleci/config.yml

```diff
@@ -111,6 +111,7 @@ jobs:
             sleep 30
       - run:
           name: Execute integration tests
+          no_output_timeout: 2m
```

Comment:
Is 2 minutes enough?

Reply:
Keep in mind it's 2 minutes without any new input on stdout/err, not 2 minutes in total. We should display steps and/or progress often. 2m looks ok-ish to me!